roles/load_aware_undeploy_trimaran/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: Delete Trimaran resources
+  shell: |
+    oc delete -f {{ trimaran_setup_config }} 2>&1
```

Comment:
no need for the `|`
let me explain how this works, it's not straightforward ;-)
```
  shell: |
    echo hello
    echo world
    echo hello \
              world
```
this is a *multi-line* string. No need for  `;`. This works and outputs:
```
hello
world
hello world
```
  shell:
    echo hello
    echo world
    echo hello \
              world
this is a *single-line* string. `;` required, as all the "lines" above are concatenated. This outputs:
```
hello echo world echo hello world
```
(not 100% sure about the last `/`, just don't do it ;-))

Reply:
can you also remove the `2>&1` and turn `shell` into `command`?